roles/tasmota-power-script/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Playbook to implement a remote startup script for users
+
+- name: install script dependencies
+  apt:
+    name:
+      - dialog
+      - curl
+    state: present
+
+- name: Make sure user is present on host
+  user:
+    name: startup_script
+    shell: /usr/bin/startup_script.sh
+    state: present
+    password: $1$ZBQbTRIM$/6OLvzsHYqHl4eszgo0hk.
```

Comment:
würde versuchen das mit SSH Key zu machen, statt passwort.

Reply:
Hier ist eine Doku dazu:
https://guacamole.apache.org/doc/gug/configuring-guacamole.html#ssh

also statt: 
![guacamole](https://user-images.githubusercontent.com/1101544/102718969-71449580-42eb-11eb-97d5-37818f883510.png)

```
password=xxxx
```

irgendwie sowas:


```
private-key=/var/lib/univention-appcenter/apps/guacamole/data/mein-ssh-key
```

Guacamole läuft als Docker.  Den Pfad habe ich aus dem Befehl: 

```
docker container inspect aa68594eb5d4
```

Die Id: *aa68594eb5d4* aus dem Befehl: 

```
docker ps
```

ganz sauber wäre es, noch das hinzufügen von Guacamole-Sessions  mit udm über ansible zu machen

Reply:
Wenn du möchtest, kann ich auch den private-key Teil in Ansible einfügen.

Reply:
Ja, steht beides auf meiner ToDo Liste. Danke für die Hinweise. Wollte nur erstmal das es überhaupt funktioniert.

Reply:
Aufgrund von allgemeinen (bekannten) Problemen bei diesem Vorgehen, habe ich es auf ein generiertes Passwort geändert.